doc/man3/EVP_PKEY_get_field_type.pod

```diff
@@ -0,0 +1,52 @@
+=pod
+
+=head1 NAME
+
+EVP_PKEY_get_field_type, EVP_PKEY_get_ec_point_conv_form - get field type
+or point conversion form of a key
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ int EVP_PKEY_get_field_type(const EVP_PKEY *pkey);
+ int EVP_PKEY_get_ec_point_conv_form(const EVP_PKEY *pkey);
+
+=head1 DESCRIPTION
+
+EVP_PKEY_get_field_type() returns the field type of the I<pkey>, if
+I<pkey>'s key type supports it.
+
+EVP_PKEY_get_ec_point_conv_form() returns the point conversion format
+of the I<pkey>, if I<pkey>'s key type supports it.
+
+=head1 NOTES
+
+Among the standard OpenSSL key types, this is only supported for EC and
+SM2 keys.  Other providers may support this for additional key types.
+
+=head1 RETURN VALUES
+
+EVP_PKEY_get_field_type() returns the field type NID or 0 on error.
```

Comment:
Perhaps list the currently valid NIDs for our own providers?

Reply:
Done